src/chapters/razor-views/template-conditionals.rst

```diff
@@ -0,0 +1,171 @@
+.. How do we conditionally display data?
+
+Conditionals in a Template
+==========================
+
+In addition to iteration, Razor can also add or remove content on a
+webpage based on certain conditions. Going back to the coffee example, we could
+generate the ordered list ONLY IF ``coffeeOptions`` contains data. If the
+list of coffee types is empty, then there is no need to include the ``<ol>`` 
+element. Instead, the template could include a ``<p>`` element with text stating 
+that there are no options to select.
+
+Display Content ``if/else``
+---------------------------
+
+To include conditional logic to display a Razor template element, we use a basic C# ``if`` clause.
+
+The general syntax for including a conditional in Razor is:
+
+.. sourcecode:: guess
+
+   @if (condition)
+   {
+      // HTML element(s) to add or additional template logic
+   }
+
+Above, ``condition`` represents any expression that can be evaluated to ``true`` or ``false`` 
+(ie, a boolean).
+
+If ``condition`` evaluates to ``true``, then Razor adds the HTML element to
+the webpage and the content is displayed in the view. If ``condition`` is
+``false``, then Razor does NOT generate the element, and the content stays
+off the page.
+
+.. admonition:: Example
```

Comment:
This kind of jumped for me. The paragraph ends with the idea that content doesn't end up on the page and then transitions to an example where ``else`` is used to make sure alternate content ends up on the page. Maybe if it transitions with some sentence that notes that we can also use a larger if/else construction to control which content appears on the page, it wouldn't read as disconnected?